www/includes/easyparliament/templates/html/misc/_register_entry.php

```diff
@@ -0,0 +1,49 @@
+<?php /** @var MySociety\TheyWorkForYou\DataClass\Regmem\InfoEntry $entry */ ?>
+
+<div class="interest-item" id="<?= $entry->comparable_id ?>">
+
+<?php if (!$entry->details->isEmpty()): ?>
+
+        <ul class="interest-details-list">
+            <?php foreach ($entry->details as $detail): ?>
+                <?php if (in_array($detail->slug, ["mysoc_summary", "mp_comment", "industry"])): ?>
+                    <?php continue; ?>
+                <?php endif; ?>
+                <?php include '_register_field.php'; ?>
+            <?php endforeach; ?>
+            <?php if ($entry->date_registered): ?>
+                <li class="registration-date">Registration Date: <?= htmlspecialchars($entry->date_registered) ?></li>
+            <?php endif; ?>
+            <?php if ($entry->date_published): ?>
+                <li class="published-date">Published Date: <?= htmlspecialchars($entry->date_published) ?></li>
+            <?php endif; ?>
+            <?php if ($entry->date_updated): ?>
+                <li class="last-updated-date">Last Updated Date: <?= htmlspecialchars($entry->date_updated) ?></li>
+            <?php endif; ?>
+        </ul>
+
+    <?php else: ?>
```

Comment:
indentation is not helpful here. And is there a reason this and the following are combined into an else if?